src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1914,7 +1914,17 @@ private AssemblySymbol GetForwardedToAssembly(string fullName, int arity, out bo
 
         internal static void CheckFeatureAvailability(Location location, MessageID feature, DiagnosticBag diagnostics)
         {
-            LanguageVersion availableVersion = ((CSharpParseOptions)location.SourceTree.Options).LanguageVersion;
+            var options = (CSharpParseOptions)location.SourceTree.Options;
```

Comment:
Is the feature flag still a parse-only option? What are we doing for features which have new semantics but no new syntax (e.g., get-only autoprop initialization in C# 6 had this property).